site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPage.tsx

```diff
@@ -28,7 +30,21 @@ export const IdpOrgSyncPage: FC = () => {
 		data: orgSyncSettingsData,
 		isLoading,
 		error,
-	} = useQuery(organizationIdpSyncSettings(isIdpSyncEnabled));
+	} = useQuery(
+		organizationIdpSyncSettings(isIdpSyncEnabled).queryKey,
+		organizationIdpSyncSettings(isIdpSyncEnabled).queryFn,
+		{
+			onSuccess: (data) => {
+				if (data?.field) {
+					setClaimField(data.field);
+				}
+			},
+		},
```

Comment:
this form is removed in newer versions of react-query, and I'm not crazy about calling `organizationIdpSyncSettings` twice. why not just
```suggestion
		} = useQuery({
			...organizationIdpSyncSettings(isIdpSyncEnabled),
			onSuccess: (data) => {
				if (data?.field) {
					setClaimField(data.field);
				}
			},
		});
```